contracts/contracts/RequestManager.sol

```diff
@@ -138,11 +139,8 @@ contract RequestManager is Ownable, LpWhitelist {
     /// cannot be used to create new requests.
     bool public deprecated;
 
-    /// The request counter, used to generate request IDs.
-    uint256 public requestCounter;
-
-    /// The claim counter, used to generate claim IDs.
-    uint256 public claimCounter;
+    /// The nonce counter, used to generate request and claim IDs.
+    uint256 public nonceCounter;
```

Comment:
Nitpick: it would be nice to have a bit more detail of the expected usage of `nonceCounter`. Currently it is not obvious whether we are supposed to read the counter and then increment it, or first increment it and then read the value.

One might also consider changing the name so it's more clear, e.g. `nextNonce` or something.